FALSEHOODS.md

```diff
@@ -118,3 +118,8 @@ surprising how many false assumptions programmers continue to make about them.
 
     In Egypt, it is common for phone numbers to be written in native digits.
 
+1.  **Phone numbers have only one prefix**
```

Comment:
By prefix, do you mean national destination code (or "area code" for geographical numbers)?

Maybe a better title would be "There is only one sequence of digits that reaches a single end-point"? Although that isn't so readable by normal people. Maybe: "There is only one area code or national destination code for a given phone number at a given time"? Or "Phone numbers have only one area code or national destination code at a given time"?